tools/madokolint.py

```diff
@@ -216,10 +260,24 @@ def check(line, lineno):
     foreach_line(path, ContextCompose(ContextAfterTitle(), ContextSkipBlocks()), check)
 
 
+def check_keywords(path):
+    def check(line, lineno):
+        for word in line.split():
+            if word not in lint_conf.keywords:
+                continue
+            category = lint_conf.keywords[word]
+            lint_state.error(
+                path, lineno, line,
+                "'{}' is a known keyword ({}), highlight it with backticks".format(word, category))
```

Comment:
Out of curiosity, does this actually verify backticks surround a keyword, or does it just detect "naked" keywords by virtue of the `split()` behavior?

Reply:
Just looks for naked keywords at the moment, so `'keyword'` will not trigger an error. I'm sure there is room for improvement :)

Reply:
OK, I was just verifying my understandinig, It's still valuable!